ui/qt-lib/src/Ariadne/UI/Qt/Widgets/Dialogs/Send.hs

```diff
@@ -196,26 +272,26 @@ runSend uiWalletFace sendInputs = do
     QDialog.Rejected -> SendCancel
 
 fillSendOptions :: Send -> IO (Validation Errors SendOptions)
-fillSendOptions Send{..} = do
-  amount <- fromString <$> QLineEdit.text amountEdit
-  address <- fromString <$> QLineEdit.text addressEdit
-
+fillSendOptions s@Send{..} = do
+  enabledReceivers <- getEnabledReceivers s
+  amounts <- mapM (\r'@Receiver{..} -> liftM2 (,) (return $ r') (fromString <$> QLineEdit.text amountEdit)) enabledReceivers
```

Comment:
Any particular reason to use `liftM2 (,)` instead of `(,) <$> ... <*> ...`? Also, since first argument is pure, that actually can be replaced with tuple secion `(r', ) <$> ...`.
Also, redundant `$`.